.mvn/extensions.xml

```diff
@@ -0,0 +1,7 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
```

Comment:
Can you give an example how we can benefit from incrementals?

Reply:
https://github.com/jenkinsci/configuration-as-code-plugin/pull/920
![image](https://user-images.githubusercontent.com/1661688/59234402-e9a5a300-8bec-11e9-8b5b-3f535939679e.png)
https://repo.jenkins-ci.org/incrementals/io/jenkins/configuration-as-code/1.21-rc815.5a901666bf93/

Reply:
The purpose is to make an internal release meaning the release that will not be a part of production?

Reply:
The purpose is to make a release I can consume from a downstream package.

So, for instance, you now depend on JCasC for testing.

Now I make a change in a pull request for JCasC.
Then I use the incremental version number here:
https://github.com/baymac/gitlab-branch-source-plugin/blob/c471569a56454fa9da36ac0fd16417eac7c67ff2/pom.xml#L25
to demonstrate that my upstream pull request works in your downstream pull request.

Before incremental, I would manually have to do a `mvn deploy`

Reply:
I get it now. Thanks.